docusaurus/docs/distributing-your-plugin.md

```diff
@@ -3,19 +3,19 @@ id: distributing-your-plugin
 title: Distributing Your Plugin
 ---
 
-When distributing a Grafana plugin either within the community or privately the plugin must be signed so the Grafana application can verify its authenticity. This can be done with the `@grafana/sign-plugin` package.
+To distribute a Grafana plugin either to the community or privately the plugin must be signed so the Grafana application can verify its authenticity. This can be done with the `@grafana/sign-plugin` package.
```

Comment:
add comma here: "privately, the plugin must be signed"

Reply:
Avoid passive tense. Change "This can be done with the..." to "You can do this with the..."